usr/lib/systemd/system/greenboot.service

```diff
@@ -0,0 +1,17 @@
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=greenboot Success Scripts Runner
+BindsTo=greenboot.target
+After=greenboot.target
+OnFailure=redboot.service
```

Comment:
@jlebon: This can be moved into the target. Where do you prefer it?

Reply:
This belongs in the target anyway, no? We should run `redboot.service` if we fail to run `check` scripts, not if we fail to run `green` scripts, right?